akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/TestKitUtils.scala

```diff
@@ -18,25 +18,27 @@ import scala.concurrent.duration.Duration
 @InternalApi
 private[akka] object ActorTestKitGuardian {
   sealed trait TestKitCommand
-  final case class SpawnActor[T](name: String, behavior: Behavior[T], replyTo: ActorRef[ActorRef[T]], props: Props) extends TestKitCommand
-  final case class SpawnActorAnonymous[T](behavior: Behavior[T], replyTo: ActorRef[ActorRef[T]], props: Props) extends TestKitCommand
+  final case class SpawnActor[T](name: String, behavior: Behavior[T], replyTo: ActorRef[ActorRef[T]], props: Props)
+      extends TestKitCommand
```

Comment:
What do we think about automatically adding line breaks like these?

Reply:
I don't like it here and it ads 4 spaces, should be 2. 
This may be configurable with `verticalMultiline.arityThreshold`

Reply:
I would prefer if it stayed on one line if within the 120 chars margin. Can't find a setting for it, do you know?

Reply:
I think it's actually doing the right thing here, the line exceeds 120 and then that is a good place to break it.

I was worried that it always breaks before `extends`

Reply:
> I don't like it here and it ads 4 spaces, should be 2. Perhaps the default is 4.
> `continuationIndent.defnSite = 2`
> ?

Bodies are indented 2, so continuations being indented 4 is a nice visual guide that it's not the body.  IMHO